src/index.html

```diff
@@ -1,13 +1,37 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container" data-qa="card">
+      <div class="mac" >
+      </div>
+      <div class="info">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
```

Comment:
```suggestion
      <h3 class="info">
        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
      </h3>
```
Це текстовий елемент. Давай використовувати семантичні теги
```suggestion
      <div class="info">
        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
      </div>
```